### PR TITLE
Feature development

### PR DESCRIPTION
**Pull request by RadovanTomik**
**What's in the PR**
* ...Added a plugin to skip checkstyle violations and modified the instructions to ease manual deployment

**How to test manually**
* ... Try the new instructions on a clean VM

### CODE REVIEW

**Review by alexanderkiel — CHANGES_REQUESTED** [review]
Please rebase onto the current develop.

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </parent>
 
   <artifactId>share-client</artifactId>
-  <version>7.4.0-SNAPSHOT</version>
+  <version>7.4.0</version>
```
Comment (TasDeniz) [inline pom.xml:14]: Please use a release version in the master branch. In the develop branch there is only a SNAPSHOT version allowed.

**Comment by RadovanTomik** [comment]
Okay I did the rebase, could you check that I haven't broken anything?

**Comment by TasDeniz** [comment]
you must resolve the pom.xml conflict

**Review by alexanderkiel — CHANGES_REQUESTED** [review]
Please remove the merge commit from develop into your feature branch, rebase all commits on the current develop branch and squash all commits into preferably one. Otherwise, I can't easily see what this PR is about. Also, please give the PR a clear, concise title without the start of "Feature".

**Pull request closed without merging**